content/francais/python-turtle/the-basics.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Les Bases - Dessiner une ligne colorée"
```

Comment:
Seems like the title was still translated here (but correctly).

Reply:
ya few files it translated really well